frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -1260,10 +1270,68 @@ https://archiveweb.page/images/${"logo.svg"}`}
       )}
       ${this.renderHelpTextCol(
         msg(
-          `Automatically click on all link-like elements. Useful for capturing in-page interactions or for clicking links without navigating away from the page.`,
+          `Automatically click on all link-like elements without navigating away from the page.`,
         ),
         false,
       )}
+      ${when(
+        this.formState.autoclickBehavior,
+        () => html`
+          ${inputCol(
+            html`<btrix-syntax-input
+              name="clickSelector"
+              label=${labelFor.clickSelector}
+              language="css"
+              value=${this.formState.clickSelector}
+              placeholder="${msg("Default:")} ${DEFAULT_AUTOCLICK_SELECTOR}"
+              disableTooltip
+              @btrix-change=${(
+                e: BtrixChangeEvent<typeof this.formState.clickSelector>,
+              ) => {
+                const el = e.target as SyntaxInput;
+                const value = e.detail.value.trim();
+
+                if (value) {
+                  try {
+                    // Validate selector
+                    this.cssParser(value);
+
+                    this.updateFormState(
+                      {
+                        clickSelector: e.detail.value,
+                      },
+                      true,
+                    );
+
+                    this.clickSelector?.removeAttribute("data-invalid");
+                    this.clickSelector?.removeAttribute("data-user-invalid");
+                  } catch {
+                    el.setCustomValidity(
+                      msg("Please enter a valid CSS selector"),
+                    );
+                  }
+                }
+              }}
+              @btrix-invalid=${() => {
+                /**
+                 * HACK Set data attribute manually so that
+                 * table works with `syncTabErrorState`
+                 *
+                 * FIXME Should be fixed with
+                 * https://github.com/webrecorder/browsertrix/issues/2497
+                 * or
+                 * https://github.com/webrecorder/browsertrix/issues/2536
+                 */
+                this.clickSelector?.setAttribute("data-invalid", "true");
+                this.clickSelector?.setAttribute("data-user-invalid", "true");
+              }}
+            ></btrix-syntax-input>`,
+          )}
+          ${this.renderHelpTextCol(
+            msg(`Customize the CSS selector used to autoclick elements.`),
```

Comment:
Accessibility: placeholders are considered purely decorative, and so usually are ignored by screen readers. Info about what the default selector is should probably be part of this help text, even if surfaced only to screen readers (via `.sr-only`).
```suggestion
            html`${msg(
                `Customize the CSS selector used to autoclick elements.`,
              )} <span class="sr-only">${msg('Defaults to "a".')}</span>`,
```